code/src/styled-components.js

```diff
@@ -0,0 +1,144 @@
+import styled from 'styled-components/macro';

```

Comment:
This is interesting with the macro - tried google the difference with or without the macro - but did not quite understand it?